<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PixGrab 2.6</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressource.qrc">
    <normaloff>:/icone/icone.png</normaloff>:/icone/icone.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Délai avant capture (secondes) :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="editDelai"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkEntiere">
      <property name="text">
       <string>Capturer l'écran entier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkHide">
      <property name="text">
       <string>Cacher cette fenêtre</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>26</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCapture">
      <property name="text">
       <string>Capturer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionQt"/>
    <addaction name="actionAbout"/>
    <addaction name="actionLicence"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionLicence">
   <property name="text">
    <string>Licence</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ressource.qrc"/>
 </resources>
 <connections/>
</ui>
